File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    let lux = 51;
    let luy = 51;
    let rdx = 0;
    let rdy = 0;
    
    for (let r = 0; r < wallpaper.length; r++) {
        for (let c = 0; c < wallpaper[0].length; c++) {
            if (wallpaper[r][c] === '#') {
                lux = Math.min(lux, r);
                luy = Math.min(luy, c);
                rdx = Math.max(rdx, r + 1);
                rdy = Math.max(rdy, c + 1);
            }
        }
    }
    
    return [lux, luy, rdx, rdy];
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux, luy = 51, 51
    rdx, rdy = 0, 0
    
    for r in range(len(wallpaper)):
        for c in range(len(wallpaper[0])):
            if wallpaper[r][c] == "#":
                lux = min(lux, r)
                luy = min(luy, c)
                rdx = max(rdx, r + 1)
                rdy = max(rdy, c + 1)
                
    return [lux, luy, rdx, rdy]